test/unitTests/build/buildExecutor.test.ts

```diff
@@ -368,4 +369,39 @@ describe('BuildExecutor', () => {
             fakedEnvironmentController.debugMode = false;
         });
     });
+
+    describe('Language Client', () => {
+        before(() => {
+            sinon.stub(LanguageClient.prototype, 'registerProposedFeatures').callsFake(() => {
+                return;
+            });
+            sinon.stub(LanguageClient.prototype, 'start').callsFake(() => {
+                return undefined;
+            });
+        });
+
+        it('Public contributor', async () => {
+            sinon.stub(fakedEnvironmentController, "userType").get(function getUserType() {

```

Comment:
same as buildController test, you need to verify that the function `registerProposedFeatures ` and `start` get called.